libs/accounts/src/lib/accounts-manager.spec.tsx

```diff
@@ -59,14 +59,17 @@ describe('AccountManager', () => {
     await waitFor(() => {
       expect(screen.getByText('No accounts')).toBeInTheDocument();
     });
-    expect(mockedUpdate({ data: [] })).toEqual(true);
-    expect(
-      mockedUpdate({ data: [{ party: { id: 't1' } }] as AccountFields[] })
-    ).toEqual(false);
-    expect(
-      mockedUpdate({ data: [{ party: { id: 't2' } }] as AccountFields[] })
-    ).toEqual(true);
-    expect(mockedUpdate({ data: [] })).toEqual(false);
-    expect(mockedUpdate({ data: [] })).toEqual(true);
+    await act(() => {
```

Comment:
This kind of looks like incorrect usage of `act`. Act should be used to wrap user interactions and `render`, though testing-library automatically does this for us. It shouldn't really be used to wrap assertions. This recently lead to broken/evergreen tests in use-environment.spec.tsx. Would be good to check that this isn't happening here